src/python/grpcio/grpc/_channel.py

```diff
@@ -99,6 +99,24 @@ def _wait_once_until(condition, until):
     else:
       condition.wait(timeout=remaining)
 
+def _check_call_error(call_error, metadata=None):
```

Comment:
Keyword arguments generally are not appropriate for private methods - make `metadata` positional?